extra/resources/coqdocjs.js

```diff
@@ -160,11 +160,11 @@ function repairDom(){
 
 function fixTitle(){
   var url = "/" + window.location.pathname;
-  var modulename = "." + url.substring(url.lastIndexOf('/')+1, url.lastIndexOf('.'));
-  modulename = modulename.substring(modulename.lastIndexOf('.')+1);
+  var basename = url.substring(url.lastIndexOf('/')+1, url.lastIndexOf('.'));
+  modulename = basename.substring(("." + basename).lastIndexOf('.'));
   if (modulename === "toc") {modulename = "Table of Contents";}
   else if (modulename === "indexpage") {modulename = "Index";}
-  else {modulename = modulename + ".v";};
+  else {modulename = basename + ".v";};
```

Comment:
Could you remove this branch and not add the ".v" extension? It's confusing to have a module name followed by .v since it's neither a file name nor a module name.

Reply:
Ok, I removed that branch (and actually all of `modulename` too, which doesn't make any sense).